wolfcrypt/src/wc_port.c

```diff
@@ -2031,6 +2031,7 @@ int wolfSSL_CryptHwMutexUnLock(void)
     }
 
 #elif defined(EBSNET)
+    #if (RTPLATFORM)
```

Comment:
You missed a few places where `RTPLATFORM` is used without checking if defined. Perhaps it is safe to assume if EBSNET is defined that `RTPLATFORM` is always defined?